services/src/main/java/org/jboss/windup/web/services/rest/UnhandledExceptionHandler.java

```diff
@@ -62,15 +62,15 @@ else if (null != findFirstOccurenceInExceptionChain("org.h2.jdbc.JdbcSQLExceptio
 
     static final Throwable findFirstOccurenceInExceptionChain(Class<? extends Throwable> type, Throwable exception)
     {
-        Throwable cause = exception;
+        Throwable cause = exception.getCause();
```

Comment:
I don't understand the purpose of this part of the change. It seems to me that there are two scenarios:

1. exception.getCause() returns null - In the previous case, this would have returned "exception"
2. exception.getCause() returns non-null and matches type.isAssignableFrom(). In this case, it seems like the previous code would have entered an infinite loop as it repeatedly calls "exception.getCause()" instead of "cause.getCause()"?!

It seems to me that the new code doesn't really fix this issue.